### PR TITLE
Disallow votes for own pics, videos and albums

### PR DESCRIPTION
**Pull request by dimuller**
This modification avoids that a member can vote for her/his own pictures, videos or albums.
I added a function **readProfileIdOf** to the **UserCoreModel**. The function retrieves the profileId of a picture, video or album from the corresponding table. Maybe you know a better location for this function.
Everything else is done in the class **RatingCoreAjax**. 

Ticket: https://github.com/pH7Software/pH7-Social-Dating-CMS/issues/763

### CODE REVIEW

File: UserCoreModel.php [hunk 1]
```diff
@@ -0,0 +1,1732 @@
+<?php
```
Comment (pH-7) [inline UserCoreModel.php:1]: Oops, looks like `UserCoreModel.php` has been added in the root directory.

Comment (dimuller) [reply UserCoreModel.php]: Oh, sorry about that! I'm not yet very familiar with Github.
I deleted the file in the root. The first committed **UserCoreModel.php** should be at the right position. Only the function **readProfileIdOf** has been added.

Comment (dimuller) [reply UserCoreModel.php]: I've seen that there has been already changes to the UserCoreModel file, so I downloaded the latest version, added the function and uploaded it again. In the hurry I must have made a mistake with the location.

Comment (polynamaude) [reply UserCoreModel.php]: @dimuller Hi !
What you shall do is for modifying in Github
1. clone repo.
2. create a branch from master
3. make modification.
4. make a git pull/fetch to update your repo
5. merge back to master
6. push back and make the PR

Comment (polynamaude) [reply UserCoreModel.php]: @dimuller Also, I think people shall be allowed to vote on their own profile, many time time also. So they can have higher rating and feel more confident of themselves.

Comment (dimuller) [reply UserCoreModel.php]: @polynamaude thanks for you guidelines. On the next occasion I will try to follow your instructions.

**Pull request closed without merging**